pb/rpc.proto

```diff
@@ -19,6 +19,8 @@ message Message {
 	optional bytes data = 2;
 	optional bytes seqno = 3;
 	repeated string topicIDs = 4;
+	optional bytes signature = 5;
```

Comment:
in light of my other comment, how about we make key come before signature, and space them out a bit (instead of 5 and 6, use 50 and 51) so future additions to the protocol don't have to come after it.

cc @Stebalien @warpfork to tell me "no, you cant just slice protobufs like that"

Reply:
the key is not included in the signature, so it can stay where it is.

Reply:
I'm not sure I can say "you can't" but I probably "wouldn't" if I could help it.  I'd be a lot more comfortable if an envelope type `{content, sig}` was added.

I feel like holistically, even across wildly different formats, people tend to rue the day they made in-band signatures.  For (distant, but again, holistically comparable) example, android putting sigs on apks *inside* the filesystem has been no end of comedic problems over the years.

Reply:
Yeah, I guess one way to do this would be to have in the top level protobuf both the existing regular unsigned messages, and a new field for signed messages.

Reply:
that also kind of disrupts the chronology of the development of the protobuf. i think we'd probably be best off keeping it as is